CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+## v0.1.0 [2025-04-02]
+
+_What's new?_
+
+- New tables added
+  - [apache_access_log](https://hub.tailpipe.io/plugins/turbot/apache/tables/apache_access_log)
+    - Query access logs with combined log format
+    - Support for file-based collection, compressed logs (gzip, zip), and AWS S3 integration
+    - Built-in queries for analyzing failed requests, large responses, and traffic patterns
```

Comment:
```suggestion
- New tables added
  - [apache_access_log](https://hub.tailpipe.io/plugins/turbot/apache/tables/apache_access_log)
```

I think we don't need the extra notes here, as most of this info is in the linked Hub doc